src/Components/WebAssembly/Sdk/src/AssetsManifestFile.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.NET.Sdk.BlazorWebAssembly
+{
+#pragma warning disable IDE1006 // Naming Styles
+    public class AssetsManifestFile
```

Comment:
These are copied pretty much verbatim from https://github.com/dotnet/aspnetcore/tree/master/src/Razor/Microsoft.NET.Sdk.Razor/src (with the namespace changes). We can remove the changes from RazorSDK once this is inserted in the .NET SDK.